### DISCUSSION TITLE
关于安全性的请教

### DISCUSSION THREAD

**Discussion by GD2021** [1Panel-dev/1Panel#4374] [Q&A]
在 VPS 安装上，安装了 1Panel 面板，并部署了一些 Docker 服务，并为各种服务配置了反代域名

🔒 仅允许域名访问服务
- 如何只允许域名访问服务，禁止 IP+端口访问服务

🕵️‍♂️ 防止被扫描发现
- 如何防止被 zoomeye 这种网站扫描到服务

**Comment by wanghe-fit2cloud** [comment]
将容器端口绑定到 127.0.0.1，然后通过 `网站` 反向代理实现域名访问
